client/src/components/shared/fetchReviews.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+const fetchReviews = (id, sort = 'helpful') => {
+  return axios.get('/reviews', {params: {
+    product_id: id,
+    count: 100000,
+    sort: sort
+  }})
+}
+
+export default fetchReviews;
```

Comment:
Moved the reviews data fetching to this separate function so I can fetch just the reviews data without having to fetch all the data. This function is called after changing sorting option, after posting a new review, or after marking a review as helpful.